docs/contributing.md

```diff
@@ -109,7 +109,7 @@ Before opening a pull request, you can preview your changes if you have set up t
 
 or if you only have Python 3 installed:
```

Comment:
Maybe this part should be removed altogether, as the QMK CLI has since become more and more of a required component.

Reply:
I've recently have to use this, and I found this particular bit of information quite useful